man/50-xdg-data-dirs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This part should be dropped I think. It's not run during build and I'd prefer not to make the strange she-bang line visible.

The same goes for `/usr/bin/env python3` in the subsequent example (not part of this PR).